### PR TITLE
Story #16: As a User, I can add my First Name, Last Name and TwitterClone Handle to my profile.

### PR DESCRIPTION
**Pull request by shrmnk**
## What
As a User, I can add my First Name, Last Name and TwitterClone Handle to my profile.

## Why

image
image

## ERD

image

Updated to include the new fields, `first_name`, `last_name` and `handle`

---

**Before submitting, check that:**

 - [X] You have written good* commit messages.
 - [X] You have squashed relevant commits together.
 - [X] You have ensured that RuboCop is passing.
 - [X] Your PR relates to one subject, with a clear title and description using complete, grammatically correct sentences.
 - [X] You have included a screenshot of the new view.

### CODE REVIEW

File: app/views/devise/registrations/edit.html.slim [hunk 1]
```diff
@@ -11,6 +11,17 @@
       .col-xs-12.col-sm-offset-2.col-sm-8.col-md-offset-4.col-md-4
         = simple_form_for(resource, as: resource_name, url: registration_path(resource_name), html: { method: :put }) do |f|
           = f.error_notification
+          
+          .form-group
+            = f.input :first_name, required: true, class: "form-control", autofocus: true
```
Comment (tbssc) [inline app/views/devise/registrations/edit.html.slim]: think `simple_form` already adds the `form-control` to the input tag. Can you verify it?

File: db/migrate/20170602035745_add_fields_to_users.rb [hunk 2]
```diff
@@ -0,0 +1,7 @@
+class AddFieldsToUsers < ActiveRecord::Migration
+  def change
+    add_column :users, :first_name, :string
+    add_column :users, :last_name, :string
+    add_column :users, :handle, :string
```
Comment (tbssc) [inline db/migrate/20170602035745_add_fields_to_users.rb]: `handle` column should be a required field. So add validations (presence and uniqueness) on the model, and `index: true, null: false` in the migration

File: app/views/devise/registrations/edit.html.slim (hunk 1 above)
Comment (shrmnk) [inline app/views/devise/registrations/edit.html.slim]: It appears so! Except when using the input-group wrapper.

File: db/migrate/20170602035745_add_fields_to_users.rb (hunk 2 above)
Comment (shrmnk) [inline db/migrate/20170602035745_add_fields_to_users.rb]: Got it!

File: app/views/devise/registrations/edit.html.slim (hunk 1 above)
Comment (tbssc) [inline app/views/devise/registrations/edit.html.slim]: even the `form-group` is already added by simple_form. So now, the actual HTML you are producing is
```slim
.form-group
  .form-group
    label
    input
```

**Comment by shrmnk** [comment]
Updated the PR with the up-to-date ERD

image

**Pull request merged**